composer.json

```diff
@@ -49,6 +49,8 @@
             "php": "5.5.9"
         }
     },
+    "prefer-stable": true,
+    "minimum-stability": "dev",
```

Comment:
This must not be forgotten when `3.1` will reach stabilization/release phase :confused: